networking/src/ApiClient.ts

```diff
@@ -624,6 +843,128 @@ type AWSTeamAnswer = {
     teamMemberAnswersId?: string | null
     confidenceLevel: ConfidenceLevel
 }
+// ~~~~~~~~~GAMETEMPLATE~~~~~~~~~~~~~~
+class GameTemplateParser {
+    static gameTemplateFromAWSGameTemplate(
+        awsGameTemplate: AWSGameTemplate
+    ): IGameTemplate {
+        let questionTemplates: IQuestionTemplate[] = awsGameTemplate.questionTemplates?.items?.filter((questionTemplate: AWSQuestionTemplate | null) => !isNullOrUndefined(questionTemplate)) || [];
+
+        const {
+            id,
+            title,
+            owner,
+            version,
+            description,
+            domain,
+            cluster,
+            grade,
+            standard,
+            phaseOneTime,
+            phaseTwoTime,
+            imageUrl,
+            createdAt,
+            updatedAt
+        } = awsGameTemplate || {}
+
+        if (isNullOrUndefined(id) ||
+            isNullOrUndefined(title) ||
+            isNullOrUndefined(owner) ||
+            isNullOrUndefined(version) ||
+            isNullOrUndefined(description) ||
+            isNullOrUndefined(domain) ||
+            isNullOrUndefined(cluster) ||
+            isNullOrUndefined(grade) ||
+            isNullOrUndefined(standard) ||
+            isNullOrUndefined(phaseOneTime) ||
+            isNullOrUndefined(phaseTwoTime) ||
+            isNullOrUndefined(imageUrl) ||
+            isNullOrUndefined(createdAt) ||
+            isNullOrUndefined(updatedAt)) {
```

Comment:
Similar to below, not all of these need to be checked here

Reply:
fixed